packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/registries/delete/delete.controller.js

```diff
@@ -0,0 +1,47 @@
+import get from 'lodash/get';
```

Comment:
**nitpick:** Can you please avoid using lodash and use native javascript capabilities like "safe traversal operator"?

Reply:
Hi @anooparveti , yes I did that, and also modify in add.controller.js that was using lodash too. 
Thx